### DISCUSSION TITLE
Installation on Windows11

### DISCUSSION THREAD

**Discussion by WangLiguo-kyrie** [marrink-lab/polyply_1.0#257] [Q&A] [answered]
I created virtual environment on Win11 by conda to install polyply, all dependencies are satisfied, but polyply cound't be run.
While the same operation was used on Linux, it worked.
Did OS affect the installation?

**Comment by fgrunewald** [comment]
In principle polyply is installable independent of the OS. However, for example number acceleration might not be available. Does the installation print any errors or what exactly fails when you run it on windows?

**Comment by WangLiguo-kyrie** [comment]
There is no error in installation. 
<img width="506" alt="image" src="https://user-images.githubusercontent.com/97430007/186124389-49706164-83de-4b65-bd7e-8903a28055e9.png">

when Running Polyply:
"Polyply 'is not an internal or external command, nor is it a executable program Or batch files".


By the way, same error encountered when installing martinize2 on Win11 (still worked on Linux)

**Comment by fgrunewald** [comment]
My guess would be that it is not correctly exposed as executable, because the library is obviously installed and there. Perhaps @pckroon has an idea what is going on here, because he uses windows more often than I do.

**Comment by pckroon** [comment]
It has to do with the PATH variable. See, e.g., also https://www.tutorialexample.com/a-simple-guide-to-add-conda-command-into-windows-path-environment-python-tutorial/

**Comment by WangLiguo-kyrie** [comment]
I think that's not the reason. Because my conda command could be used normally and the conda path in this tutorial was added successfully before.
<img width="311" alt="image" src="https://user-images.githubusercontent.com/97430007/186167328-865a7d8c-dd55-4263-8705-e55f8dfee5ba.png">
Besides, I tried to execute these two programs directly in conda/Scripts folder, the error above still existed.

**Comment by pckroon** [comment]
It is absolutely an issue with your PATH variable. Conda might be in your path, but martinize2/polyply (obviously) are not. I'm not sure the main entrypoints get installed in Scripts or somewhere else.

**Comment by WangLiguo-kyrie** [comment, marked as answer]
I fould when I run
`python martinize2`
or
`python polyply`
it worked.

In linux, we only need run `martinize2` or `polyply`.
lol

**Reply by thangckt** [reply]
this does not work with me

**Comment by fgrunewald** [comment]
Humm that is indeed interesting. If no one objects I will transfer the issue onto the discussion page, because to me this is less related to polyply installation than the individual setup of the conda environment on windows machines. But this could still be useful for others
